Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx

```diff
@@ -1183,6 +1209,19 @@ class TPCTimeSeries : public Task
           typeSide = 1;
         }
 
+        // check for TOF and propagate TPC track to TOF cluster
+        bool hasTOFCluster = (idxTPCTrackToTOFCluster[iTrk] != -1);
+        auto tofCl = hasTOFCluster ? tofClusters[idxTPCTrackToTOFCluster[iTrk]] : o2::tof::Cluster();
+
+        float tpcYatTOF = 0;
+        float tpcZatTOF = 0;
+        if (hasTOFCluster) {
+          if (trackTmp.rotate(o2::math_utils::sector2Angle(tofCl.getSector())) && propagator->propagateTo(trackTmp, tofCl.getX(), false, mMaxSnp, mFineStep, mMatType)) {
```

Comment:
Hello @shahor02 
we wanted to get for each TPC track that has a TOF cluster assigned the TPC track propagated to the TOF cluster position and then compare the propagated TPC track position with the TOF cluster position.
Can you take a look if I did it properly? 
I am also not quite sure if the propagation is correctly done. I assumed here that the `tfCl.getX()` returns the local X position of the cluster.

Reply:
One should not use the TPC innermost  track param for propagation to TOF, instead use (since you don't use errors)
```
o2::track::TrackPar trackTmpOut(tracksTPC[iTrk].getParamOut());
...
```